.platform.app.yaml

```diff
@@ -21,6 +21,10 @@ dependencies:
 # by default.
 hooks:
     build: |
+      # Generate the CLI command index page.
+      curl -sS https://platform.sh/cli/installer | php
+      platform list --format md > src/overview/cli-commands.md
```

Comment:
This is generating an empty file, although because it happens in the build env I'm not entirely sure why.  This part works when I run `platform build` locally.  @pjcdawkins, please advise.

Reply:
You'd need `source .environment` between those two lines: the installer adds `platform` to the PATH via the `.environment` file for new shells, but it can't change the PATH in the running shell.